backend/medtagger/api/auth/business.py

```diff
@@ -0,0 +1,45 @@
+"""Module responsible for business logic in all Auth endpoint."""
+from flask_security import login_user, logout_user
+from werkzeug.security import generate_password_hash, check_password_hash
+
+from medtagger.api import InvalidArgumentsException
+from medtagger.database.models import User
+from medtagger.repositories.roles import RolesRepository
+from medtagger.repositories.users import UsersRepository
+
+
+def create_user(email: str, password: str, first_name: str, last_name: str) -> int:
```

Comment:
Please update docstring with given arguments. More about it can be found here: https://stackoverflow.com/a/24385103/2839291 (reST format)

Reply:
Is this really necessary in that example? These arguments are straightforward. I don't quite see the sense in updating doc string with:  `:param email: user's email address` and so on.

Reply:
Fixed